src/graphql/getters/getSeasonBadges/mappers.ts

```diff
@@ -0,0 +1,82 @@
+import { BadgeVariant } from './types';
+import { Season } from '../../../hooks/season/types';
+
+export const BADGE_TYPE_BADGE_VARIANT_MAP: Record<string, BadgeVariant> = {
+  // season 1
+  '12': 'fixedTrader',
```

Comment:
are these id's for each badge from the BE?

Reply:
yes! I sat with @ioanaroceanu2000 and we did the mapping. So all you need to do in future is just extend this map

Reply:
amazing!